File: CS265/zeta.cpp
#include <math.h>
#include <stdio.h>
#include <iostream>

using namespace std;

int main( int argc, char **argv) {

int x;
int n;

cout << "Enter Value for x: ";
cin >> x;
cout << "Enter Value for n: ";
cin >> n;
 
float total = 0;

for (int i=0; i<n; i++){
	total = total + pow((1.0/(float(i)+1.0)),float(x));
}

cout << "z(" << x << ")=" << total << " when approximated at n=" << n << "\n";
return 0;
}

File: CS265/zeta.py
import sys

# sys.argv[1] == x, sys.argv[2] == n
# x = int(sys.argv[1])
# n = int(sys.argv[2])
# total = 0
#
# for i in range(n):
#     total = total + (1/(i+1))**x
#
# print('z(', n, ')=', round(total,7), 'when approximated at n=', n)
print(sys.argv[0])
print(sys.argv[1])
print(len(sys.argv))

A1 = list()
for i in range(size):
    A1.append(sys.argv[i+1])